src/ansys/hps/client/__version__.py

```diff
@@ -29,6 +29,10 @@
 # major, minor, patch
 __version__ = importlib_metadata.version("ansys-hps-client")
 
+__company__ = "Ansys Switzerland GmbH"
+
+__company_short__ = "Ansys"
+
 # this is only a convenience to default the version
 # of Ansys simulation applications in PyHPS examples
-__ansys_apps_version__ = "2024 R1"
+__ansys_apps_version__ = "2024 R2"
```

Comment:
bumping apps version is release related, should not be done in this PR

Reply:
reverted the change